test/integration/install-variants_test.js

```diff
@@ -29,7 +31,7 @@ suite('install-variants', function() {
           assert.fail(err);
           done(err);
         }
-        runCommand(binPath, ['install', '--variants', '--offline'], {cwd: tmpPath}).then(() => {
+        runCommand(binPath, ['install', '--variants'], {cwd: tmpPath}).then(() => {
```

Comment:
It's not env variables apparently, no bower env variables found: https://travis-ci.org/Polymer/polymer-cli/jobs/202150103#L2339